README.rst

```diff
@@ -250,6 +250,10 @@ Choose from the list of available rules:
 
   - ``allow_single_line_closure`` (``bool``): whether single line lambda notation
     should be allowed; defaults to ``false``
+  - ``position_after_control_structures`` (``'same'``): whether the opening brace
```

Comment:
Looks like there an issue with README helper: the option has two allowed values but only one is listed here. This is not related to this PR as the `position_after_functions_and_oop_constructs` option has the same issue. I'll investigate that.